### PR TITLE
Convert some more ops to use Output<Node> inputs

### PR DESCRIPTION
**Pull request by diyessi**

### CODE REVIEW

File: src/ngraph/op/experimental/quantized_avg_pool.cpp [hunk 1]
```diff
@@ -36,14 +38,20 @@ op::QuantizedAvgPool::QuantizedAvgPool(const shared_ptr<Node>& arg,
 {
     constructor_validate_and_infer_types();
 
-    if (arg->get_element_type() != element::u8 && arg->get_element_type() != element::i8)
+    if (arg.get_element_type() != element::u8 && arg.get_element_type() != element::i8)
```
Comment (aprocter) [inline src/ngraph/op/experimental/quantized_avg_pool.cpp]: This check was copied and pasted into `validate_and_infer_types()`. Delete it here?

File: src/ngraph/op/experimental/quantized_conv_bias.hpp [hunk 2]
```diff
@@ -27,25 +27,28 @@ namespace ngraph
...
 
             const Strides& get_window_movement_strides() const { return m_window_movement_strides; }
             const Strides& get_window_dilation_strides() const { return m_window_dilation_strides; }
             const CoordinateDiff& get_padding_below() const { return m_padding_below; }
             const CoordinateDiff& get_padding_above() const { return m_padding_above; }
             const Strides& get_data_dilation_strides() const { return m_data_dilation_strides; }
-            std::shared_ptr<Node> get_bias() { return get_argument(2); }
-            std::shared_ptr<Node> get_filters() { return get_argument(1); }
-            std::shared_ptr<Node> get_data_batch() { return get_argument(0); }
+            Output<Node> get_bias() { return input(2).get_source_output(); }
```
Comment (aprocter) [inline src/ngraph/op/experimental/quantized_conv_bias.hpp]: I'm glad that this worked. I was afraid to touch it before. :)

File: src/ngraph/op/experimental/quantized_max_pool.cpp [hunk 3]
```diff
@@ -22,20 +22,22 @@
...
+    : Op({arg})
     , m_window_shape(window_shape)
     , m_window_movement_strides(window_movement_strides)
     , m_padding_below(padding_below)
     , m_padding_above(padding_above)
 {
     constructor_validate_and_infer_types();
 
-    if (arg->get_element_type() != element::u8 && arg->get_element_type() != element::i8)
+    if (arg.get_element_type() != element::u8 && arg.get_element_type() != element::i8)
```
Comment (aprocter) [inline src/ngraph/op/experimental/quantized_max_pool.cpp]: Should this go to `validate_and_infer_types()`?

File: src/ngraph/op/select.cpp [hunk 4]
```diff
@@ -26,10 +26,10 @@
 using namespace std;
 using namespace ngraph;
 
-op::Select::Select(const shared_ptr<Node>& arg0,
-                   const shared_ptr<Node>& arg1,
-                   const shared_ptr<Node>& arg2)
-    : Op("Select", check_single_output_args({arg0, arg1, arg2}))
+const string op::Select::type_name{"Select"};
+
+op::Select::Select(const Output<Node>& arg0, const Output<Node>& arg1, const Output<Node>& arg2)
```
Comment (aprocter) [inline src/ngraph/op/select.cpp]: Who named these arguments???

File: src/ngraph/op/experimental/quantized_max_pool.cpp (hunk 3 above)
Comment (diyessi) [reply src/ngraph/op/experimental/quantized_max_pool.cpp]: Yes, and dozens of others. Another PR.

**Pull request merged**